integration/stability/soak_parallel_rm.sh

```diff
@@ -137,12 +137,12 @@ go() {
 }
 
 kill_all_containers() {
-	present=$(sudo ctr c list -q | wc -l)
-	if ((${present})); then
-		sudo ctr tasks kill $(sudo ctr task ls -q)
-		sudo ctr tasks rm -f $(sudo ctr task list -q)
-		sudo ctr c rm $(sudo ctr c list -q)
-	fi
+	for container in $(sudo ctr containers list -q); do
+		sudo ctr tasks kill "$container"
+		# Give task a second to die if required
```

Comment:
One second wait seems short. Did you try on a system with a load level of 50?

Reply:
I can test that, but I'd like to mention that we previously ran with no wait whatsoever. I think `ctr tasks kill` is officially even just supposed to return after kill completion, but it seems we sometimes don't _quite_ hit that.

Reply:
Yes, this works even under full load.